main.gd

```diff
@@ -2,19 +2,34 @@
 class_name MainScript
 extends Node
 
+@onready var piece_scene = load("res://piece.tscn")
+
+var initial_piece
+
 
 ## Called when the node enters the scene tree for the first time.
 func _ready():
-	## Enables unhandled input (gameplay input) processing by the _unhandled_key_input virtual function.
-	set_process_unhandled_key_input(true)
+	randomize()  ## randomize rng seed
```

Comment:
nit: Implementation comments should be prefixed with one `#`—`##` should only be used to provide method descriptions. I double-checked the docs and saw that `##` for the implementation comments do nothing.

For consistency, comments should have proper capitalization and punctuation; see [Google's Cpp Style Guide](https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar).

```suggestion
	randomize()  # Randomize rng seed.
```